src/latex/conway-ledger.tex

```diff
@@ -10,37 +10,43 @@
 \input{preamble}
 
 \begin{document}
+
 \title{Formal Specification of the Cardano Ledger for the Conway era}
 
-\author{
-   Andre Knispel \\ {\small \texttt{andre.knispel@iohk.io}} \\
-   William DeMeo \\ {\small \texttt{william.demeo@iohk.io}} \\
-   Joosep Jääger \\ {\small \texttt{joosep.jaager@iohk.io}} \\
-   }
```

Comment:
I'm not sure why we have this short list of authors here.  Afaik, this was never discussed.

I think it's safer to simply include a list of contributors that names everyone who contributed to the document, in alphabetical order.

The list included here, and especially in the `PDF.lagda` file (for `cardano-ledger.pdf`), probably needs to be expanded a bit.

Reply:
I think \author is the appropriate LaTeX command to use. However it should look like:
```latex
\author{
   Andre Knispel \\ {\small \texttt{andre.knispel@iohk.io}} 
   \and
   William DeMeo \\ {\small \texttt{william.demeo@iohk.io}}
   \and
   Joosep Jääger \\ {\small \texttt{joosep.jaager@iohk.io}}
}
```
We might want to define a macro to typeset email addresses